### PR TITLE
{tools}[intel/2022a] HPL v2.0.15 w/ CUDA 11.7.0

### PR DESCRIPTION
**Pull request by boegel**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
node3308.joltik.os - Linux RHEL 8.6, x86_64, Intel(R) Xeon(R) Gold 6242 CPU @ 2.80GHz (cascadelake), 1 x NVIDIA Tesla V100-SXM2-32GB, 530.30.02, Python 3.6.8
See https://gist.github.com/boegel/0909c083249c22f14897802c728ac80f for a full test report.

File: easybuild/easyconfigs/h/HPL/HPL-2.0.15-intel-2022a-CUDA-11.7.0.eb [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+
+homepage = 'https://www.netlib.org/benchmark/hpl/'
+description = """HPL is a software package that solves a (random) dense linear system in double precision (64 bits)
+ arithmetic on distributed-memory computers. It can thus be regarded as a portable as well as freely available
+ implementation of the High Performance Computing Linpack Benchmark."""
+
+toolchain = {'name': 'intel', 'version': '2022a'}
+toolchainopts = {'usempi': True}
+
+# download via https://developer.nvidia.com/computeworks-developer-exclusive-downloads
```
Comment (branfosj) [inline easybuild/easyconfigs/h/HPL/HPL-2.0.15-intel-2022a-CUDA-11.7.0.eb:16]: Please use `download_instructions` instead here.

File: easybuild/easyconfigs/h/HPL/HPL-2.0.15-intel-2022a-CUDA-11.7.0.eb [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+sources = ['hpl-%%(version_major_minor)s_FERMI_v%s.tgz' % local_patch_version]
+patches = ['HPL_parallel-make.patch']
+checksums = [
+    {'hpl-2.0_FERMI_v15.tgz': '16dd21af22fced613923a6cb09805d63952fc9c9f278cb7582061a43fe39a408'},
+    {'HPL_parallel-make.patch': '2a5bf9c4f328049828ddecec7ba3f05a9e25d236f4212747c53bd22fea80c5e6'},
+]
+
+# fix Make dependencies, so parallel build also works
```
Comment (branfosj) [inline easybuild/easyconfigs/h/HPL/HPL-2.0.15-intel-2022a-CUDA-11.7.0.eb:24]: This comment is in the wrong place.

**Review by branfosj — CHANGES_REQUESTED** [review]

File: easybuild/easyconfigs/h/HPL/HPL-2.0.15-intel-2022a-CUDA-11.7.0.eb (hunk 2 above)
Comment (branfosj) [inline easybuild/easyconfigs/h/HPL/HPL-2.0.15-intel-2022a-CUDA-11.7.0.eb:24]:
```suggestion
download_instructions = """Download CUDA Accelerated Linpack from
https://developer.nvidia.com/computeworks-developer-exclusive-downloads"""
sources = ['hpl-%%(version_major_minor)s_FERMI_v%s.tgz' % local_patch_version]
# fix Make dependencies, so parallel build also works
patches = ['HPL_parallel-make.patch']
checksums = [
    {'hpl-2.0_FERMI_v15.tgz': '16dd21af22fced613923a6cb09805d63952fc9c9f278cb7582061a43fe39a408'},
    {'HPL_parallel-make.patch': '2a5bf9c4f328049828ddecec7ba3f05a9e25d236f4212747c53bd22fea80c5e6'},
]

```

Comment (branfosj) [reply easybuild/easyconfigs/h/HPL/HPL-2.0.15-intel-2022a-CUDA-11.7.0.eb]: done in 07a000a

File: easybuild/easyconfigs/h/HPL/HPL-2.0.15-intel-2022a-CUDA-11.7.0.eb (hunk 1 above)
Comment (branfosj) [reply easybuild/easyconfigs/h/HPL/HPL-2.0.15-intel-2022a-CUDA-11.7.0.eb]: done in 07a000a

**Comment by branfosj** [comment]
Test report by @branfosj
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
bear-pg0208u03a - Linux RHEL 8.6, x86_64, Intel(R) Xeon(R) Platinum 8360Y CPU @ 2.40GHz (icelake), 1 x NVIDIA NVIDIA A100-SXM4-40GB, 520.61.05, Python 3.6.8
See https://gist.github.com/branfosj/51d6bd699505275c4392ee8702baa986 for a full test report.

**Review by branfosj — APPROVED** [review]

**Comment by branfosj** [comment]
Going in, thanks @boegel!

**Pull request merged**